### PR TITLE
Remove slashes from namespaces

### PR DESCRIPTION
**Pull request by gadelkareem**
fixes https://github.com/NYTimes/drone-gke/issues/102

### CODE REVIEW

**Comment by yunzhu-li** [comment]
Hi @gadelkareem, thanks for your contribution! Would you mind to provide more context of your issue and use case for #102?

I wonder if the plugin is the ideal tool for processing values vs always pass them as-is to `kubectl`. For example, you could also use Drone to do some [string operations](https://0-8-0.docs.drone.io/substitution/#string-operations-reference) before they are passed into the plugin. I'm curious what @tonglil thinks.

**Comment by gadelkareem** [comment]
@yunzhu-li I didn't know about Drone string operations, thanks for the tip. I think this might be useful still as a quick fix for someone using branch name like 'feature/something' as a namespace. Or we can just update the docs at  https://github.com/NYTimes/drone-gke/blob/master/DOCS.md

**Comment by gadelkareem** [comment]
Updated!

File: main.go [hunk 1]
```diff
@@ -46,6 +47,7 @@ kind: Namespace
 metadata:
   name: %s
 `
+var invalidNameRegex = regexp.MustCompile(`[^a-z\.\-]+`)
```
Comment (tonglil) [inline main.go]: so close, but just missing 0-9

Comment (gadelkareem) [reply main.go]: fixed!

Comment (gadelkareem) [reply main.go]: added!

**Comment by tonglil** [comment]
Thanks, sorry for the delay!

**Pull request merged**